contracts/lz/interfaces/ILayerZeroUserApplicationConfig.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.5.0;
+
+interface ILayerZeroUserApplicationConfig {
```

Comment:
Please, instead of creating a new file, let's import `ILayerZeroUserApplicationConfig` adding LayerZero repo as dependency